README.md

```diff
@@ -143,9 +143,12 @@ Request [devnet sol](https://faucet.solana.com/) as necessary.
 cd example/program
 cargo build-sbf --sbf-out-dir ./target
 solana config set -ud
-solana program deploy --program-id target/fibonacci_verifier_contract-keypair.json target/fibonacci_verifier_contract.so
+solana program deploy --program-id target/fibonacci_verifier_contract-keypair.json target/fibonacci_verifier_contract.so --compute-unit-limit 300000
```

Comment:
i think i might need to make another change in the example, to bump the solana runtime's CU's

Reply:
nvm i tried it out im tripping